xbmc/interfaces/builtins/GUIBuiltins.cpp

```diff
@@ -22,6 +22,9 @@
 #include "input/WindowTranslator.h"
 #include "input/actions/ActionTranslator.h"
 #include "messaging/ApplicationMessenger.h"
+#if defined(TARGET_WINDOWS)
```

Comment:
please avoid using ifdefs in common sources

Reply:
cannot be easily avoided

Reply:
but possible. nobody said that it's easy. we did a lot of work to clean up the code from ifdefs, please don't produce them again

Reply:
Done